core/src/main/java/org/apache/accumulo/core/util/Validators.java

```diff
@@ -177,7 +176,7 @@ void warnTooLong(String s) {
     if (id == null)
       return Optional.of("Table id must not be null");
     @SuppressWarnings("deprecation")
-    TableId replicationId = ReplicationTable.ID;
+    TableId replicationId = org.apache.accumulo.core.replication.ReplicationTable.ID;
```

Comment:
Curios if there was a specific reason you dropped the import in favor of the fully qualified name for ReplicationTable?

Reply:
Probably to avoid deprecation warnings.